packages/core/tocco-ui/src/Table/TableHeader.js

```diff
@@ -5,17 +5,23 @@ import {dragAndDrop, resize} from 'tocco-util'
 import {columnPropType, dataPropType} from './propTypes'
 import ResizingController from './ResizingController'
 import SortingState from './SortingState'
-import {StyledDnD, StyledDraggable, StyledTableHead, StyledTableHeaderCell, StyledTableRow} from './StyledTable'
+import {
+  StyledHeaderContent,
+  StyledDnD,
+  StyledDraggable,
+  StyledTableHead,
+  StyledTableHeaderCell,
+  StyledTableRow
+} from './StyledComponents'
 
 const ThContent = ({column, data}) =>
   column.HeaderRenderer ? (
     <column.HeaderRenderer column={column} data={data} />
   ) : (
-    <div
-      dangerouslySetInnerHTML={{__html: column.label}}
-      title={column.label}
-      style={{width: '100%'}} // div should take up full available width, otherwise content cannot be right aligned
-    />
+    <StyledHeaderContent title={column.label}>
+      {column.label}
```

Comment:
dangerouslySetInnerHTML is required so https://toccoag.atlassian.net/browse/TOCDEV-2061